### ISSUE TITLE
MangaLife Downloading/Loading Improvement 

### ISSUE THREAD

**Issue by Bluesnake-42** [tachiyomiorg/tachiyomi-extensions#13880]
### Source name and language

MangaLife

### Describe your suggested feature

The loading and downloading of chapter pages is extremely slow compared to most other extensions I use.
When a Manhua or manga chapter only has a few long pages this is not an issue but when the chapters have 100+ pages/images, it is impossible to read the chapter without having to wait for it load. the only option is to download the chapter but it takes so long to download it. i recently added a manhwa that has 60 chapter with each chapter having 150+ pages, its been downloading for over 12 hours and is not even half way done.
I understand the extension might work this way in order to avoid being blocked from the website but please check if the extension's download/loading can be optimized in any way.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by curche** [comment]
This won't be changed because it is intended behavior
There's too many users using the extension, updating manga, downloading chapters which itself has caused the website collapsing in the past. This subsequently leads to Tachiyomi being blocked frequently

The only options were to either remove the extension or throttle it enough that the website can handle it

>  i recently added a manhwa that has 60 chapter with each chapter having 150+ pages, its been downloading for over 12 hours and is not even half way done.

if you're downloading 60 chapters each with 150+ pages, you'd have definitely got blocked by the website itself....
clear your download queue completely, wait a while, read through below links, and try again later

For everyone's benefit, please go through
 - [troubleshooting](https://tachiyomi.org/help/guides/troubleshooting/)
 - [suggestions](https://tachiyomi.org/help/faq/#library)
And spread out your manga across multiple websites
